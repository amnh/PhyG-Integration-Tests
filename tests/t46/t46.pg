--build(distance, nj), inputting fasta and binary files
set(seed:1634561640)
read(fasta:"chel.fas", "chel_binary.fas")
build(distance, nj)
report("t46.csv", data, overwrite)
report("t46.tre", newick, graphs, overwrite)