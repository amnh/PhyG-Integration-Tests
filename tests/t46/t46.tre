(Alentus:20.0,(Aportus:4.0,(Rhiphicephalus:16.0,((Vonones:12.0,((Amblypygid:8.0,(Hypochilus:2.0,Thelichoris:3.0)HTU18:7.0)HTU21:3.0,(Mastigoproctus:8.0,(Chanbria:8.0,Limulus:10.0)HTU20:3.0)HTU23:3.0)HTU24:3.0)HTU25:2.0,((Centruoides:10.0,(Hadrurus:9.0,Pauroctonus:2.0)HTU19:8.0)HTU22:6.0,((Americhernus:29.0,Gea:13.0)HTU26:15.0,(Artemia:16.0,Thermobius:10.0)HTU27:4.0)HTU28:2.0)HTU29:3.0)HTU31:4.0)HTU30:10.0)HTU32:18.0)HTU17[520.0];
