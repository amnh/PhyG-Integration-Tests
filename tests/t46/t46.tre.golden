(Alentus:50.0,(Aportus:26.5,(Rhiphicephalus:32.5,((Vonones:21.5,((Amblypygid:17.0,(Hypochilus:9.0,Thelichoris:10.0)HTU18:11.0)HTU21:9.0,(Mastigoproctus:17.5,(Chanbria:19.0,Limulus:19.0)HTU20:9.0)HTU23:8.5)HTU24:9.5)HTU25:7.0,((Centruoides:18.5,(Hadrurus:18.0,Pauroctonus:10.0)HTU19:17.0)HTU22:13.0,((Americhernus:43.5,Gea:27.5)HTU26:30.5,(Artemia:34.5,Thermobius:31.5)HTU27:17.0)HTU28:11.0)HTU29:9.0)HTU31:15.0)HTU30:18.0)HTU32:43.0)HTU17[520.0];
